extension/snippets/dvc-yaml.code-snippets

```diff
@@ -0,0 +1,55 @@
+{
+  "Get Started with DVC pipelines": {
+    "prefix": "dvc-help",
+    "body": [
+      "# Snippets are provided by the DVC for VS Code extension to help with building dvc.yaml pipelines.",
+      "# These snippets are all prefixed with dvc-.",
+      "# Type dvc- in the editor to see the list of available snippets.",
+      "# Each snippet is designed to give pointers on basic usage.",
```

Comment:
Minor, should we combine sentences on lines 6 and 8? For example, `These snippets, all prefixed with dvc-, are designed to give pointers on basic usage`.